engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -29,6 +29,8 @@ case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
                                     locallyQualifiedInputs: CallInputs = Map.empty,
                                     abortRegistrationFunction: Option[AbortRegistrationFunction] = None) extends JobDescriptor[BackendCallKey] {
 
+  def call = key.scope
```

Comment:
Yeah I think when I finally decided to do it I had already changed 95% of them to `key.scope`...
On a different note I was wondering why all of those parameterless methods are `def`s and not `val`s here, since this is `case class` and not a `trait` ?